src/Stories.js

```diff
@@ -35,7 +35,7 @@ var Stories = React.createClass({
     }
   },
 
-  componentWillMount() {
+  componentDidMount() {
```

Comment:
more out of interest, but why is this required? Is this what gets around that weird React error I was getting?